packages/ios-client/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@fishjam-cloud/ios-client",
   "scripts": {
-    "lint": "swift-format format -i -r ./Sources/ --configuration swift-format-config.json",
-    "lint:check": "swift-format lint -r -s ./Sources/ --configuration swift-format-config.json"
+    "lint": "sh ./scripts/lint.sh",
+    "lint:check": "sh ./scripts/lint_check.sh"
```

Comment:
I think that for iOS both `lint` and `lint:check` should do the same thing. There is now (easy) way to do dry-run for files, so hopefully this will catch potential issues)